Resizin/Classes/ResizinSettings.swift

```diff
@@ -59,6 +59,36 @@ public struct ResizinSettings {
         case right = "90"
         case down = "180"
     }
+    
+    /// Returns the image transformed into the selected format
+    public enum OutputFormat {
+        /// Returns image in original format
+        case original
```

Comment:
Well I am wondering if `original` is desired option? This could mean that on older systems than iOS 14, we can receive webp, which is unsupported...are we okay with that possibility? What do you think @fortmarek? Maybe I would go for `static var default: OutputFormat` which would be used if `outputFormat` is not specified in concrete call.

Reply:
ahh, forgot about this case - but I don't think we are able to tell in advance if the image's format is `webp`? In other words, what would be in the static var that would prohibit the clients from trying to download webp images on < iOS 14 ?

Reply:
Well idea was to have `.png` there or the default option could be mandatory argument when calling `setupSharedManager` when configuring Reqres...